processor/solarwindsprocessor/internal/host_attributes_decorator_test.go

```diff
@@ -22,11 +22,11 @@ import (
 	"go.uber.org/zap"
 )
 
-func TestApplyAttributes_HostId_NonCloudHostWithFallbackHostId(t *testing.T) {
+func TestApplyAttributes_HostId_NonCloudHostWithoverrideHostId(t *testing.T) {
```

Comment:
```suggestion
func TestApplyAttributes_HostId_NonCloudHostWithOverrideHostId(t *testing.T) {
```

Other cases also below